src/wallet/rpc/wallet.cpp

```diff
@@ -220,7 +220,7 @@ static RPCHelpMan loadwallet()
                 "\nNote that all wallet command-line options used when starting bitcoind will be"
                 "\napplied to the new wallet.\n",
                 {
-                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The wallet directory or .dat file."},
+                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The path of the wallet directory (or a legacy wallet's .dat file). Takes an absolute path or a path relative to the default wallet directory."},
```

Comment:
We have already [updated](https://github.com/bitcoin/bitcoin/pull/26990#discussion_r1762156524) `<filename>` to `<walletname>` in a few places (`bitcoin-cli.cpp` and `wallet/rpc/utils.cpp`), shouldn't be this the case as well?
nit:
```suggestion
                    {"walletname", RPCArg::Type::STR, RPCArg::Optional::NO, "The path to the wallet directory (absolute or relative to the wallets directory), or a legacy wallet's .dat file (within the wallets directory). The default wallets directory is the 'wallets' folder inside the data directory, unless the `-walletdir` option is specified."},
```

Reply:
Sounds fair to me - made this change 👍

Reply:
My bad, my suggestion was more for discussion as I did it on top of @achow101's one. The tests can be updated but I see we could be breaking things on users end if they pass named args (`filename=`) as in `wallet_startup,py`.

Reply:
Perhaps that can be a separate change to this then? Fixed so that change is not breaking.